### DISCUSSION TITLE
How to install on Docker-based Home Assistant?

### DISCUSSION THREAD

**Discussion by asgrim** [hassio-addons/addon-node-red#1432] [Q&A] [answered]
Hey folks. Struggling to find out how to actually install this. I tried step 1 of [the docs](https://github.com/hassio-addons/addon-node-red/blob/main/node-red/DOCS.md#installation), but it fails straight away;

> This redirect is not supported by your Home Assistant installation. It needs either the Home Assistant Operating System or Home Assistant Supervised installation method. For more information, see the [documentation](https://www.home-assistant.io/installation).

So then I wonder - is this a [HACS](https://hacs.xyz/) enabled project? I try add `https://github.com/hassio-addons/addon-node-red` but it doesn't work either:

> The repository does not seem to be a integration, but an add-on repository. HACS does not manage add-ons.

I'm running HA using Docker. I haven't used Node RED before, so wanted to try it out.

Is there any documentation/info on how to install Node RED in Home Assistant when you're not using the above install methods please?

**Comment by frenck** [comment, marked as answer]
> So then I wonder - is this a [HACS](https://hacs.xyz/) enabled project? I try add https://github.com/hassio-addons/addon-node-red but it doesn't work either:

HACS does not provide add-ons.

> Is there any documentation/info on how to install Node RED in Home Assistant when you're not using the above install methods please?

Consult the Node-RED documentation in that case. Add-ons are not available for your installation type.
